.github/workflows/ci.yml

```diff
@@ -13,6 +13,7 @@ on:
 jobs:
   lint_test:
     runs-on: ubuntu-latest
+    timeout-minutes: 10
```

Comment:
Will it take 10 mins?
Lets reduce it, and at later point when we need more time we can always update
I think 5 should be good enough for now?
What do you think. 
How much time does the build takes currently

Reply:
Yup 5 sounds good, its taking around 2m45secs